src/test/concurrency/transaction_context_test.cpp

```diff
@@ -11,14 +11,25 @@
 #include "concurrency/transaction_context.hpp"
 #include "concurrency/transaction_manager.hpp"
 #include "operators/abstract_read_write_operator.hpp"
+#include "operators/delete.hpp"
+#include "operators/get_table.hpp"
+#include "operators/validate.hpp"
 #include "types.hpp"
 #include "utils/assert.hpp"
 
 namespace opossum {
 
 class TransactionContextTest : public BaseTest {
+ public:
+  std::string table_name;
```

Comment:
What do you think about the following?
```suggestion
  static constexpr auto table_name = "test_table";
```